charts/kubernetes-agent/values.yaml

```diff
@@ -163,6 +163,18 @@ scriptPods:
     clusterRole:
       rules: []
 
+  deploymentTarget:
```

Comment:
These are missing comments. See the other image options for examples

Reply:
added comments

Reply:
These are at the wrong level, they should be above the `image` node (see the output)

Reply:
resolved